### ISSUE TITLE
rustbuild: "changelog-seen = 1" breaks config.toml

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang/rust#77105]
````
warning: x.py has made several changes recently you may want to look at
help: consider looking at the changes in `src/bootstrap/CHANGELOG.md`
note: to silence this warning, add `changelog-seen = 1` to `config.toml`
````
So I skipped through the changelog and added `changelog-seen = 1` to the end of my config toml but this broke `x.py`:

````
failed to parse TOML configuration 'config.toml': unknown field `changelog-seen`, expected one of `sign-folder`, `gpg-password-file`, `upload-addr`, `src-tarball`, `missing-tools` for key `dist` at line 522 column 1
````

**Comment by matthiaskrgr** [comment]
Apparently the script wants it at the beginning of the toml file, the warning message should says so :roll_eyes:
